CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+# Contributing to kubernetes-deploy
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to kubernetes-deploy. Please take a moment to read through them before submitting your first PR.
+
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [High-level design decisions](#high-level-design-decisions)
+  * [Feature acceptance policies](#feature-acceptance-policies)
+  * [Adding a new resource type](#contributing-a-new-resource-type)
+
+[Development](#development)
+  * [Setup](#setup)
+  * [Running the test suite locally](#running-the-test-suite-locally)
+  * [Releasing a new version (Shopify employees)](#releasing-a-new-version-shopify-employees)
+  * [CI (External contributors)](#ci-external-contributors)
+## Code of Conduct
+
+This project and everyone participating in it are governed by the [Code of Conduct](https://github.com/Shopify/kubernetes-deploy/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable
+behavior to [kubernetes-deploy@shopify.com](mailto:kubernetes-deploy@shopify.com).
+
+## Maintainers
+
+This project is currently under the stewardship of the Production Platform group at Shopify. The two primary maintainers are @knverey and @dturn; their approval is generally required for all significant feature proposals and code architecture changes.
```

Comment:
Does this mean the approval of both parties, or is one sufficient? This point could be clearer.

Reply:
I'll update  to `at least one` and include that two people must approve all non-trivial PRs